packages/api/src/services/library_item.ts

```diff
@@ -133,37 +120,35 @@ const buildWhereClause = (
       case InFilter.SUBSCRIPTION:
         queryBuilder
           .andWhere("NOT ('library' ILIKE ANY (library_item.label_names))")
-          .andWhere({
-            subscription: Not(IsNull()),
-            archivedAt: IsNull(),
-          })
+          .andWhere('library_item.archived_at IS NULL')
+          .andWhere('library_item.subscription IS NOT NULL')
```

Comment:
nice, seems like that ILIKE above was a bug.

Reply:
sql looks okay. let me test it in the db

Reply:
verified in database and it is not a bug